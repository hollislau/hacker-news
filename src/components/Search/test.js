import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Search from '.';

describe('Search', () => {
  const props = {
    value: 'search',
    onChange: () => {},
    onSubmit: () => {}
  }

  it('renders', () => {
    const div = document.createElement('div');

    ReactDOM.render(<Search { ...props }>Search</Search>, div);
  });

  test('snapshots', () => {
    const component = renderer.create(<Search { ...props }>Search</Search>);
    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
});
