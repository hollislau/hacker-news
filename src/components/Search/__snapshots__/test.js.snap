exports[`Search snapshots 1`] = `
<form
  onSubmit={[Function]}>
  <input
    onChange={[Function]}
    type="text"
    value="search" />
  <button
    type="submit">
    Search
  </button>
</form>
`;
